modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/prepare/ExplainPlan.java

```diff
@@ -36,10 +36,12 @@ public class ExplainPlan implements QueryPlan {
 
     private final PlanId id;
     private final MultiStepPlan plan;
+    private final ParameterMetadata parameterMetadata;
 
-    ExplainPlan(PlanId id, MultiStepPlan plan) {
+    ExplainPlan(PlanId id, MultiStepPlan plan, ParameterMetadata parameterMetadata) {
```

Comment:
when `parameterMetadata` for explain may be different from one returned by the plan it's meant to explain?

Reply:
Thanks. It never does.